src/main/resources/openapi/audio-requests.yaml

```diff
@@ -0,0 +1,96 @@
+openapi: 3.0.1
+
+servers:
+  - url: http://localhost:4550/
+
+info:
+  description: Modernised DARTS (Digital Audio Recording and Transcription Service).
+  version: "0.0.1"
+  title: Modernised DARTS
+
+paths:
+  /audio-requests:
+    get:
+      security:
+        - bearerAuth: [ ]
+      tags:
+        - Audio Requests
+      summary: Users can view a summary of their audio requests and status
+      operationId: getYourAudio
+      description: '"Your audio" requests for a user to view whether Current / Expired'
+      parameters:
+        - in: header
+          name: user_id
+          schema:
+            type: integer
+          description: The user_account primary key
+          required: true
+        - in: query
+          name: expired
+          schema:
+            type: boolean
+          description: A way to filter whether Current / Expired
```

Comment:
maybe it should say something like `if true, will retrieve expired audio requests only, if false will get current requests only.`